src/cpu/aarch64/matmul/jit_int8_matmul.cpp

```diff
@@ -81,6 +81,7 @@ using namespace nstl;
 
 using namespace data_type;
 
+template <cpu_isa_t isa>
```

Comment:
We don't need this templating in this PR, but I know that you are about to enable sve_128, so if that is imminent, I'm not going to give you the extra work of taking this out...